source/manual/alerts/runs-rake-task-search-api.html.md

```diff
@@ -0,0 +1,25 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Runs a rake task on Search API that generates the sitemap files
```

Comment:
Can I suggest "Generate sitemaps" as the title?

Reply:
This is an alert page documentation, so I've copied the alert as it appears in Icinga. I think this is the convention for all documentation under `manual/alerts`.

Reply:
OK, thank you for explaining!